xml-conduit/test/main.hs

```diff
@@ -141,9 +141,9 @@ combinators = C.runResourceT $ P.parseLBS def input C.$$ do
         P.force "need child2" $ P.tagNoAttr "child2" $ return ()
         P.force "need child3" $ P.tagNoAttr "child3" $ do
             x <- P.contentMaybe
-            liftIO $ x @?= Just "combine <all> &content"
+            liftIO $ x @?= Just "\160 combine <all> &content"
   where
-    input = L.concat
+    input = L.unlines
```

Comment:
I changed this to unlines to a) test handling new-lines within the document, and b) also so the the error messages would be easier to read (since they have useful line numbers instead of a single unhelpful column number)